test/db/cmd/cmd_rop

```diff
@@ -336,19 +478,16 @@ Gadget size: 12
   0x0000012e           0e946b04  call 0x8d6
 Gadget size: 12
 
-  0x0000013a               6de6  ldi r22, 0x6d
   0x0000013c               71e0  ldi r23, 0x01
   0x0000013e               8fe1  ldi r24, 0x1f
   0x00000140               92e0  ldi r25, 0x02
   0x00000142           0e946b04  call 0x8d6
-Gadget size: 12
+Gadget size: 10
 
```

Comment:
Updated this TC as it is a duplicate and not printed out, hence the Gadget starts from `13c`